src/Integrations/Integrations/PDO/PDOIntegration.php

```diff
@@ -72,29 +69,24 @@ public function init()
                 $span->meta = [
                     'db.rowcount' => $retval->rowCount(),
                 ];
-                PDOIntegration::storeStatementFromConnection($this, $retval);
```

Comment:
@bwoebi about your comment on this. In this case works because it returns an "executed" statement and we only trace `execute` on `PDOStatement` (which is not called in this case). But thinking more about it, I'd say that it makes sense to keep it in sync in any case. Say that in 1 month we want to trace `fetchAll()` (which might not make sense since it is only an in-memory operation), then we would have a bug in case we do not assert all the fields.